aeron-driver/src/main/java/io/aeron/driver/DriverConductor.java

```diff
@@ -1790,15 +1868,74 @@ private NetworkPublication newNetworkPublication(
         }
     }
 
+    private  String enrichNetworkPublicationChannelUri(
+        final ChannelUri channelUri,
+        final int streamId,
+        final UdpChannel udpChannel,
+        final SendChannelEndpoint channelEndpoint,
+        final FlowControl flowControl,
+        final PublicationParams params)
+    {
+        // todo
+
+        final ChannelUriStringBuilder uriStringBuilder = new ChannelUriStringBuilder(channelUri);
+
+        configureUriStringBuilderWithPublicationParams(params, uriStringBuilder);
+
+
+        uriStringBuilder.streamId(streamId);
+
+
+        uriStringBuilder.flowControl(flowControl.getClass().getName());
+        final String ccStr = udpChannel.channelUri().get(CONGESTION_CONTROL_PARAM_NAME);
+        uriStringBuilder.congestionControl(null == ccStr ? "static" : ccStr);
```

Comment:
The `cc` part can be removed as it is configured on a subscription side.

Reply:
Done